frontend/src/components/regions/UserFeedbackChart/Presentation.js

```diff
@@ -8,11 +8,11 @@ import {
   Tooltip,
   Legend,
 } from "recharts";
-import orange from "@material-ui/core/colors/orange";
-import green from "@material-ui/core/colors/green";
-import red from "@material-ui/core/colors/red";
-import blue from "@material-ui/core/colors/blue";
-import { data } from "./CardDataStats";
+// import orange from "@material-ui/core/colors/orange";
+// import green from "@material-ui/core/colors/green";
+// import red from "@material-ui/core/colors/red";
+// import blue from "@material-ui/core/colors/blue";
+// import { data } from "./CardDataStats";

```

Comment:
Hi @kaleem99 

Please rather remove unused code rather than commenting it out

Reply:
> Hi @kaleem99
> 
> Please rather remove unused code rather than commenting it out

Thanks, I made the requested changes also when Sbonelo was referring to that the code will need to be later changed for the sake of dry code, I checked with him and he said that it won't able to be changed now since the Other PR that has the dry code has not yet been merged with develop branch.

Reply:
Can't you and @Sbonelo01  work towards the same base, change your base branch not to be develop?